rotkehlchen/db/ens.py

```diff
@@ -86,3 +87,28 @@ def update_values(
                 mappings_to_send[address] = name
 
         return mappings_to_send
+
+    def get_last_avatar_update(self, ens_name: str) -> Timestamp:
+        """
+        Returns the timestamp when the avatar for the given ens name was updated last time.
+        May raise:
+        - InputError if given `ens_name` is not in `ens_mappings` table
+        """
+        with self.db.conn.read_ctx() as cursor:
+            result = cursor.execute(
+                'SELECT last_avatar_update FROM ens_mappings WHERE ens_name = ?',
+                (ens_name,),
+            ).fetchone()
+
+        if result is None:
+            raise InputError('Provided ens name is not being tracked')
+
+        return Timestamp(result[0])
+
+    def set_last_avatar_update(self, ens_name: str, timestamp: Timestamp) -> None:
```

Comment:
I am pretty sure this is an unneeded function.Delete it and move its code to the caller.